Refactor/19 (#52)
trulysimple/tsargp: packages/docs/pages/docs/library/formatter.mdx

### Indentation level

The `indent` property specifies the indentation levels (in number of terminal columns) of help entry
columns. It has the following set of optional properties:

- `names` - level of indentation for the option names (non-negative, defaults to `2{:ts}`)
- `param` - level of indentation for the option parameter (non-negative if `paramAbsolute` is
  `true{:ts}`, defaults to `2{:ts}`)
- `descr` - level of indentation for the option description (non-negative if `descrAbsolute` is
  `true{:ts}`, defaults to `2{:ts}`)

> 

Author: ```suggestion
```

- `paramAbsolute` whether the indentation level for the option parameter should be relative to the
  beginning of the line instead of the end of the option names (defaults to `false{:ts}`)
- `descrAbsolute` whether the indentation level for the option description should be relative to the
  beginning of the line instead of the end of the option parameter (defaults to `false{:ts}`)

### Line breaks

The `breaks` property specifies the number of line breaks to insert before help entry columns. It
has the following set of optional properties:

- `names` - line breaks to insert before the option names (defaults to `0{:ts}`)
- `param` - line breaks to insert before the option parameter (defaults to `0{:ts}`)
- `descr` - line breaks to insert before the option description (defaults to `0{:ts}`)

### Hidden columns

The `hidden` property can be used to select help entry columns that should _not_ be displayed. It
has the following set of optional properties:

> - `names` - whether option names should not be displayed (defaults to `false{:ts}`)
> - `param` - whether option parameters should not be displayed (defaults to `false{:ts}`)
> - `descr` - whether option descriptions should not be displayed (defaults to `false{:ts}`)

Author: ```suggestion
- `names` - whether option names should be omitted (defaults to `false{:ts}`)
- `param` - whether option parameters should be omitted (defaults to `false{:ts}`)
- `descr` - whether option descriptions should be omitted (defaults to `false{:ts}`)
```

### Help items

The `items` property specifies the kinds of items that should be displayed in option descriptions,

> and in what order. It is an array whose values can be one of the enumerated values from `HelpItem`:

Author: ```suggestion
and in what order. It is an array whose values can be one of the enumerators from `HelpItem`:
```
